lib/media/drm_engine.js

```diff
@@ -685,7 +685,12 @@ shaka.media.DrmEngine = class {
     // Create a config entry for each key system.
     for (const info of allDrmInfo) {
       const config = {
-        // Ignore initDataTypes.
+        /**
+         * Non-Chromium Edge throws an error when requesting a
+         * com.microsoft.playready.recommendation media key system access
+         * that doesn't have initDataTypes
+         */
+        initDataTypes: ['cenc'],
```

Comment:
❓ I guess that's not the good way to do it... We might need to wait for the first `encrypted` event to get the correct initDataType to specify here... Any ideas on that matter ?

@OrenMe @theodab